sections/futures/Trade/MarketsDropdown.tsx

```diff
@@ -62,19 +62,22 @@ type MarketsDropdownProps = {
 	mobile?: boolean;
 };
 
+const getMinDecimals = (asset: string) =>
```

Comment:
I'm nots sure this is actually needed now we use suggested

Reply:
True. Removed.